<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findBook</class>
 <widget class="QMainWindow" name="findBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>查找书籍</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>911</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>查找所有书</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按类别查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按书名查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按出版社查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按年份查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按作者查询</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按价格查询</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="word1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>借书</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
